src/components/CreateSite/CreateSiteModal.jsx

```diff
@@ -107,38 +134,59 @@ const CreateSiteModal = ({ areaId }) => {
             <div className="input-area">
               <Col md={5}>
                 <label htmlFor="site-name">
-                  Name
+                  Name<span style={{ color: '#e32' }}>*</span>
                   <input
                     type="text"
                     className="form-control"
                     name="siteName"
-                    placeholder="placeholder"
+                    placeholder="i.e. Lakeview Middle School"
                     {...register('siteName')}
                   />
                 </label>
                 <label htmlFor="address-street">
-                  Address Line
+                  Street Address<span style={{ color: '#e32' }}>*</span>
                   <input
                     type="text"
                     className="form-control"
                     name="addressStreet"
-                    placeholder="placeholder"
+                    placeholder="ie 123 Playa Dr"
                     {...register('addressStreet')}
                   />
                 </label>
+                <label htmlFor="apt-suite-etc">
+                  Apt, suite, etc
+                  <input
+                    type="text"
+                    className="form-control"
+                    name="aptSuiteEtc"
+                    placeholder="Apt 208"
+                    {...register('aptSuiteEtc')}
+                  />
+                </label>
                 <div className="input-fields-coalesce-wrapper">
                   <label htmlFor="address-city">
-                    City
+                    City<span style={{ color: '#e32' }}>*</span>
                     <input
                       type="text"
                       className="addr-small-field form-control"
                       name="addressCity"
-                      placeholder="placeholder"
+                      placeholder="Irvine"
                       {...register('addressCity')}
                     />
                   </label>
+                  <label htmlFor="address-state">
```

Comment:
🚫 **[eslint]** <jsx-a11y/label-has-associated-control> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>A form label must be associated with a control.